x-pack/plugins/maps/public/layers/sources/es_source/es_source.js

```diff
@@ -126,7 +129,14 @@ export class AbstractESSource extends AbstractVectorSource {
       allFilters.push(getTimeFilter().createFilter(indexPattern, searchFilters.timeFilters));
     }
 
+    const SearchSource = getSearchSourceType({
```

Comment:
Can you move this boilerplate code to maps/kibana_service and expose via helper method.

Reply:
sure